dll/win32/shell32/wine/classes.c

```diff
@@ -326,10 +326,10 @@ BOOL HCR_GetIconA(LPCSTR szClass, LPSTR szDest, LPCSTR szName, DWORD len, int* p
 	  RegCloseKey(hkey);
 	}
 
-    if (ret)
-        TRACE("-- %s %i\n", szDest, *picon_idx);
-    else
-        TRACE("-- not found\n");
+        if (ret)
+            TRACE("-- %s %i\n", szDest, *picon_idx);
+        else
+            TRACE("-- not found\n");
```

Comment:
Does this fix an actual warning?

Reply:
Looks also that this file mixes tabs/spaces.